[DOCS-1609] Document activity dashboard for Multi-tenant Cloud (#1447)
wandb/docs: content/en/guides/hosting/monitoring-usage/org_dashboard.md

  default:
    identifier: org_dashboard
    parent: monitoring-and-usage
title: View organization dashboard
title: View organization activity
---

{{% alert color="secondary" %}}
Organization dashboard is available only with [Dedicated Cloud]({{< relref "/guides/hosting/hosting-options/dedicated_cloud.md" >}}) and [Self-managed instances]({{< relref "/guides/hosting/hosting-options/self-managed.md" >}}).

> {{% /alert %}}

Reviewer: ~Do we want both dedicated & SaaS sites to point to the same docs, ie this page?~

Yes is the answer -- I appreciate that you added the subsections per deployment type!

    * **Active**: User has accepted the invite and created an account.
    * **-**: The user was previously active but has not been active in the last 6 months.
    * **Deactivated**: Admin has revoked access of the user.
1. To see details about a user's last activity, hover your mouse over the **Last Active** field for the user.  A tooltip appears that shows when the user was added and how many total days the user has been active.

    A user is _active_ if they:
    - log in to W&B.
    - view any page in the W&B App.
    - log runs.
    - use the SDK to track an experiment.
    - interact with the W&B Server in any way.
{{% /tab %}}

{{% tab header="Multi-tenant SaaS Cloud" value="saas" %}}
1. Navigate to the [**Members** page](https://wandb.ai/account-settings/wandb/members/). This page lists all users, along with data about each user.
1. To sort the list by user status, click the **Last Active** column label. Each user's status is one of the following:

    * **Invite pending**: Admin has sent invite but user has not accepted invitation. 
    * **Active**: User has accepted the invite and created an account.

>     * **-**: The user was previously active but has not been active in the last 6 months.

Reviewer 2: Maybe add a space before the colon? (Or describe that it's a dash?) 

It looks like a mistake in the docs.
<img width="204" height="41" alt="Screenshot 2025-07-22 at 4 06 42 PM" src="https://github.com/user-attachments/assets/2d63d7ef-ecbe-4b1f-9fa6-7aecf4c5fe4d" />

>     * **-**: The user was previously active but has not been active in the last 6 months.

Author: ```suggestion
    * `-`: A hyphen indicates that the user was previously active but has not been active in the last 6 months.
```

    * **Deactivated**: Admin has revoked access of the user.
1. To see details about a user's last activity, hover your mouse over the **Last Active** field for the user.  A tooltip appears that shows when the user was added and how many total days the user has been active.

>     A user is _active_ if they perform any auditable action scoped to the organization. For a full list, refer to [Actions]({{< relref "/guides/hosting/monitoring-usage/audit-logging.md#actions" >}}) in the Audit Logging page.

Reviewer: The exact address didn't resolve when added to `docs.wandb.ai`. For whatever reason, navigating to the page/header myself, the address shows `audit-logging/#actions`

Author: These docs are not yet merged, so you need to use the HTML preview links provided in the PR description, until the PR actually merges. Unless I am misunderstanding you.

Author: The full URL for the Actions link should be https://docs.wandb.ai/guides/hosting/monitoring-usage/audit-logging/#actions. I verified that the Action link works in the HTML preview: https://docs-1609.docodile.pages.dev/guides/hosting/monitoring-usage/org_dashboard/

Reviewer: amazing thanks!

{{% /tab %}}
{{< /tabpane >}}

## Export user details

{{< tabpane text=true >}}
{{% tab header="Dedicated or Self-managed" value="dedicated" %}}
From the **Users** tab, you can export details about how your organization uses W&B in CSV format.

1. Navigate to the **Organization Dashboard** at `https://<org-name>.io/org/dashboard/`. Replace `<org-name>` with your organization name. The **Users** tab opens by default.
1. Click the action `...` menu next to the **Invite new user user** button.

> 1. Click **Export as CSV**. The CSV files that downloads lists details about each user of an organization, such as their user name and email address, the time they were last active, their roles, and more.

Reviewer 2: ```suggestion
1. Click **Export as CSV**. The downloaded CSV file lists details about each user of an organization, such as their user name and email address, the time they were last active, their roles, and more.
```

{{% /tab %}}

{{% tab header="Multi-tenant SaaS Cloud" value="saas" %}}
1. Navigate to the [Members page](https://wandb.ai/account-settings/wandb/members/).
1. Click the action `...` menu next to the search field.

> 1. Click **Export as CSV**. The CSV files that downloads lists details about each user of an organization, such as their user name and email address, the time they were last active, their roles, and more.

Reviewer 2: ```suggestion
1. Click **Export as CSV**. The downloaded CSV file downloads lists details about each user of an organization, such as their user name and email address, the time they were last active, their roles, and more.
```

{{% /tab %}}
{{< /tabpane >}}

## View active users over time
This section shows how to get an aggregate view of how many users have been active over time.

{{< tabpane text=true >}}
{{% tab header="Dedicated or Self-managed" value="dedicated" %}}

> Use the plots in the **Activity** tab to get an aggregate view of how many users have been active over time.

Reviewer 2: Add a sentence where the Activity tab is.

Reviewer 2: @mdlinville Did you miss this one?

Author: I don't think so? Isn't it covered by lines 74-5?

> 1. Click the **Activity** tab.

Author: ```suggestion
1. To access the **Organization Dashboard**, navigate to `https://<org-name>.io/org/dashboard/`. Replace `<org-name>` with your organization name.
1. Click the **Activity** tab.
```

1. The **Total active users** plot shows how many unique users have been active in a period of time (defaults to 3 months).
1. The **Users active over time** plot shows the fluctuation of active users over a period of time (defaults to 6 months). Hover your mouse over a pointo to see the number of users on that date.

To change the period of time for a plot, use the drop-down. You can select:
- Last 30 days
- Last 3 months
- Last 6 months
- Last 12 months
- All time

{{% /tab %}}
{{% tab header="Multi-tenant SaaS Cloud" value="saas" %}}

> Use the plots in the **Organization Dashboard** to get an aggregate view of how many users have been active over time:

Reviewer 2: Add a line where the org dashboard is? It's mentioned in a Dedicated tab above, but users on the Multi-tenant SaaS Cloud route might not see it.

> Use the plots in the **Organization Dashboard** to get an aggregate view of how many users have been active over time:

Author: ```suggestion
Use the plots in the **Activity Dashboard** to get an aggregate view of how many users have been active over time:

1. Click the user profile icon at the top right.
1. Under **Account**, click **Users**.
1. View the Activity Panel above the list of users. It shows:
```

- The **Active user count** badge shows how many unique users have been active in a period of time (defaults to 3 months). A user is _active_ if they perform any auditable action scoped to the organization. For a full list, refer to [Actions]({{< relref "/guides/hosting/monitoring-usage/audit-logging.md#actions" >}}) in the Audit Logging page.

> - The **Weekly active users** plot charts how many users have been active in the last week.

Author: ```suggestion
- The **Weekly active users** plot charts how many users have been active over the period of time.
```

> - The **Active user count** badge shows how many unique users have been active in a period of time (defaults to 3 months). A user is _active_ if they perform any auditable action scoped to the organization. For a full list, refer to [Actions]({{< relref "/guides/hosting/monitoring-usage/audit-logging.md#actions" >}}) in the Audit Logging page.
> - The **Weekly active users** plot charts how many users have been active over the period of time.
> - The **Most active user** leaderboard ranks the top ten most active users by how many days they were active over the period of time, as well as when they were most recently active.

Author: ```suggestion
  - The **Active user count** badge shows how many unique users have been active in a period of time (defaults to 3 months). A user is _active_ if they perform any auditable action scoped to the organization. For a full list, refer to [Actions]({{< relref "/guides/hosting/monitoring-usage/audit-logging.md#actions" >}}) in the Audit Logging page.
  - The **Weekly active users** plot charts how many users have been active over the period of time.
  - The **Most active user** leaderboard ranks the top ten most active users by how many days they were active over the period of time, as well as when they were most recently active.
```

> - The **Most active user** leaderboard ranks the top ten most active users by how many days they were active over a period of time (defaults to 3 months), as well as when they were most recently active.

Reviewer: All of these widgets will be hooked up to the same date-range picker. It's accurate to say "defaults to 3 months", but maybe not necessary to repeat.

> - The **Most active user** leaderboard ranks the top ten most active users by how many days they were active over a period of time (defaults to 3 months), as well as when they were most recently active.

Author: ```suggestion
- The **Most active user** leaderboard ranks the top ten most active users by how many days they were active over the period of time, as well as when they were most recently active.
```

Reviewer: The original copy was actually perfect! We could probably trim the `(defaults to 3 months)` piece, but that's definitely the most accurate description.

> To adjust the span of time the plots show, click the date picker in the top right. You can choose 7, 30, or 90 days. The default date range is 30 days. All of the plots share the same time range and update automatically.

Author: ```suggestion
1. To adjust the span of time the plots show, click the date picker in the top right. You can choose 7, 30, or 90 days. The default date range is 30 days. All of the plots share the same time range and update automatically.
```

> To adjust the span of time the plots show, click the date picker in the top right. You can choose 7, 30, or 90 days, or select a custom range. All of the plots share the same time range and update automatically.

Reviewer: We've deprioritized `custom` for this iteration. That leaves us with `7, 30 or 90 days`.

> To adjust the span of time the plots show, click the date picker in the top right. You can choose 7, 30, or 90 days, or select a custom range. All of the plots share the same time range and update automatically.

Author: ```suggestion
To adjust the span of time the plots show, click the date picker in the top right. You can choose 7, 30, or 90 days. The default date range is 30 days. All of the plots share the same time range and update automatically.
```
